README.md

```diff
@@ -21,6 +21,31 @@ Currently, I run this model's server locally on my computer with every PDF I can
 4. Setup the `.env` file using the `.env.example` file.
 5. `yarn start`!
 
+### docker setup
+1. Make sure you have docker and docker-compose installed on your system.
+2. ONE TIME ONLY: give executable permission to `scripts` folder
+    ```bash
+    chmod +x ./scripts
+    ```
+3. Run following command for building and starting docker containers
+    ```bash
+    yarn docker:up
+    ```
+
+    App should be running [http://localhost:8080/](http://localhost:8080/).
+    And database should be visible [http://localhost:8081/](http://localhost:8081/)
+
+4. To run migrations
+    ```bash
+    yarn docker:migrate up # or "down"
+    ```
+5. To stop containers
+    ```bash
+    yarn docker:down
+    ```
+
+TODO: Create windows scripts
```

Comment:
what do we need to do to complete this step?

Reply:
I don't have windows system to check if .bat scripts are working or not. That's why they are in TODO.